.github/workflows/wda-package.yml

```diff
@@ -57,3 +71,106 @@ jobs:
         uses: actions/upload-artifact@v3.1.0
         with:
           path: "${{ env.ZIP_PKG_NAME_TVOS }}"
+
+
+  for_simulator_devices_x86_64:
+    needs: [host_machine]
+    name: Build WDA for simulator iOS and tvOS devices - x86-64
+    runs-on: ${{ needs.host_machine.outputs.host }}
+
+    env:
+      ZIP_PKG_NAME_IOS_SIM_X86_64: "WebDriverAgentRunner-Build-Sim-x86-64.zip"
+      ZIP_PKG_NAME_TVOS_SIM_X86_64: "WebDriverAgentRunner_tvOS-Build-Sim-x86-64.zip"
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - uses: maxim-lobanov/setup-xcode@v1
+        with:
+          xcode-version: "${{ env.XCODE_VERSION }}"
+      - name: Build iOS for simulator for x86_64
+        run: sh $GITHUB_WORKSPACE/Scripts/ci/build.sh
```

Comment:
I think it makes sense to combine build and zip into a single script as you anyway need to run them both. Then put matrix parameters into the script as each build could be done in parallel

Reply:
yea, still tweaking them